rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -354,6 +357,53 @@ void buildMigrationTransaction_withAFederationWithEnoughUTXOs_beforeRSKIP376_sho
         assertEquals(BTC_TX_VERSION_1, migrationTransactionUnsigned.getVersion());
     }
 
+    @Test
+    void createPegoutTransaction_with210Inputs_with50Outputs_shouldHaveASizeAboveMaximumBtcStandardSize() {
```

Comment:
The test naming is confusing to me. The name says you are testing the `createPegoutTransaction` method, but it is a test helper method, so I guess what is being tested is `calculateSignedSegwitBtcTxVirtualSize`. Then, why have these tests in `ReleaseTransactionBuilderTest`? According to what I see, these tests are using nothing from it. I would rather move them to `BridgeUtilsTest`, or better yet, to their own test class. Wdyt?